primap2/tests/test_data_format.py

```diff
@@ -45,7 +45,7 @@ def test_valid_ds_pass(self, any_ds, caplog):
     def test_time_dimension_for_metadata(self, opulent_processing_ds, caplog):
         opulent_processing_ds["Processing of CO2"] = opulent_processing_ds[
             "Processing of CO2"
-        ].expand_dims(dim={"time": np.array(["2020", "2021"], dtype=np.datetime64)})
+        ].expand_dims(dim={"time": np.array(["2020", "2021"], dtype="datetime64[ns]")})
```

Comment:
What is the reason to pass dtype as a string?

Reply:
if you want to specify nanosecond resolution for the dtype, it is not possible to write `np.datetime64[np.ns]` or something like that, I think. Anyway, it was easier to figure out how to specify the nanosecond resolution for the dtype as a string.